programmers/난이도별/level02.최댓값과_최솟값/rockmiin_최댓값과최솟값.py

```diff
@@ -0,0 +1,6 @@
+def solution(s):
+    s=list(map(int, s.split()))
```

Comment:
```suggestion
    s = map(int, s.split())
```
map이 원래 list를 반환합니다!

![image](https://user-images.githubusercontent.com/18749057/91376811-ec73ce80-e858-11ea-9c83-aaeb02acfe48.png)

Reply:
![스크린샷 2020-08-27 오전 11 36 17](https://user-images.githubusercontent.com/52374789/91377076-889dd580-e859-11ea-9ec1-3ae49700987b.png)
![스크린샷 2020-08-27 오전 11 35 48](https://user-images.githubusercontent.com/52374789/91377081-8c315c80-e859-11ea-9429-16ec5807e9bf.png)
list를 사용하지 않으면 이렇게 반환 되더라구요 ! 객체의 주소를 반환해서 사용을 할 수가 없던데 따로 사용하는 방법이 있을까요 ?